.github/.license_check.ini

```diff
@@ -0,0 +1,29 @@
+# Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
+        3-clause bsd
+        apache
+        apache 2.0
+        apache software
+        apache software license
+        apache-2.0
+        bsd
+        bsd license
+        GNU Lesser General Public License v2 or later (LGPLv2+)
```

Comment:
Should we move gpl license to unauthorized license?
        GNU Lesser General Public License v2 or later (LGPLv2+)
        gpu lgpl
        lgpl with exceptions or zpl

Reply:
LGPL is allowed